src/relay/op/tensor/transform.cc

```diff
@@ -2142,7 +2142,9 @@ Expr MakeSplit(Expr data,
 TVM_REGISTER_GLOBAL("relay.op._make.split")
 .set_body([](const TVMArgs& args, TVMRetValue* rv) {
     if (args.type_codes[1] == kDLInt) {
-      *rv = MakeSplit(args[0], tir::make_const(DataType::Int(64), int64_t(args[1])), args[2]);
+      *rv = MakeSplit(args[0],
+                      tir::make_const(DataType::Int(32), static_cast<int>(args[1])),
```

Comment:
Without this change, all tests in `tests/python/relay/test_pass_combine_parallel_dense.py` fail

Reply:
can we comment about why?

Reply:
Is it because the transformed pass uses int32 to construct the attributes?

Reply:
Yeah, the transformed graph uses int32. The other way is probably to find out where it is transformed and use int64 there.

Reply:
we can go with this fix for now and check back later